### PR TITLE
Fix Domiciliary Care discrepancy. 

### LINKED ISSUE #7538: Date of Domiciliary Care Commenced is missing in Treatment Summary
**Describe the bug**

- [ ] When we generate treatment summary for a domiciliary care patient, the date of commencement is missing in the generated report
image

- [ ] domiciliary care patients, the commenced date is shown as admission on badge in consultation page
image

**To Reproduce**

1. Go to patients consultation tab of a Domiciliary patient
2. Click on manage patients button and click treatment summary

**Expected behavior**

- [ ] in treatment summary report of domiciliary patients, replace the "date of admission" field with "date of domiciliary care commenced".
- [ ] replace the badge "admission on" with "commenced on"for domiciliary patient

### PR DESCRIPTION
**Pull request by AshrafMd-1**
## Proposed Changes

- Fixes #7538 
- Changed the badge `admission on` to `commenced on` in patient info card for domiciliary patients
- Changed the field from `date of admission` to `date of domiciliary care commenced` in treatment summary report for domiciliary patients

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by AshrafMd-1** [comment]
@nihal467 can you check once

**Comment by AshrafMd-1** [comment]
@rithviknishad can you check once

File: src/Components/Patient/PatientInfoCard.tsx [hunk 1]
```diff
@@ -401,7 +401,9 @@ export default function PatientInfoCard(props: {
                       <span className="flex">
                         {consultation?.encounter_date && (
                           <div>
-                            Admission on:{" "}
+                            {consultation.suggestion === "DC"
+                              ? "Commenced on: "
+                              : "Admitted on: "}{" "}
```
Comment (Ashesh3) [inline src/Components/Patient/PatientInfoCard.tsx]:
```suggestion
                              : "Admitted on: "}
```

We already have a space after `on: ` so let's not have 2 spaces.

**Review by Ashesh3 — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**